src/GlobalStyle.js

```diff
@@ -0,0 +1,22 @@
+import { createGlobalStyle } from "styled-components";
```

Comment:
Is this styling that can be used anywhere in the application?

Reply:
So that is just so we can add styling to the HTML and the Body - styling that will be common across the entire website (e.g. 100% height and width on HTML, font used across the website, etc.).